mindmeld/cli.py

```diff
@@ -445,6 +446,36 @@ def setup_blueprint(ctx, es_host, skip_kb, blueprint_name, app_path):
         ctx.exit(1)
 
 
+@module_cli.command('convert', context_settings=CONTEXT_SETTINGS)
+@click.pass_context
+@click.option('-d', '--df', is_flag=True, help="Convert a Dialogflow project")
+@click.option('-r', '--rs', is_flag=True, help="Convert a Rasa project")
+@click.argument('project_path', required=True, type=click.Path(exists=True))
+@click.argument('mindmeld_path', required=False)
+def convert(ctx, df, rs, project_path, mindmeld_path=None):
+    """Converts a Rasa or DialogueFlow project to a MindMeld project"""
+    try:
+        project_path = os.path.abspath(project_path)
+        mindmeld_path = os.path.abspath(mindmeld_path or "converted_app")
```

Comment:
Why is `converted_app ` here?

Reply:
To give a default location for the converted app in case the user does not specify

Reply:
Got it, please fix the linting issues and merge.